appserver/ejb/ejb-http-remoting/admin/src/main/java/fish/payara/ejb/http/admin/EjbInvokerService.java

```diff
@@ -118,4 +124,14 @@ public void event(EventListener.Event event) {
             }
         }
     }
+
+    @Override
+    public UnprocessedChangeEvents changed(PropertyChangeEvent[] events) {
+        List<UnprocessedChangeEvent> unchangedList = new ArrayList<>();
+        for (PropertyChangeEvent event : events) {
+            unchangedList.add(new UnprocessedChangeEvent(event, "EJB Invoker configuration changed:" + event.getPropertyName()
```

Comment:
You need some extra checks in here, as this is too broad.
Simply enabling or disabling the service now makes the server say "requires restart", when it doesn't need it for that.